workflows/n_t.snake

```diff
@@ -357,25 +349,29 @@ rule all_plot:
         #f2 = ne_files_smcpp,
         f3 = ne_files_msmc,
     output:
-        output_dir + f"/Results/all_estimated_Ne_{selection}.pdf"
+        output_dir + "/plots/{demog}/{chrms}/{dfes}/{annots}/all_estimated_Ne.pdf"
```

Comment:
cant we have the final pdf with all those names included in the name, such as:
f'all_estimated_Ne_{demog}/{chrms}/{dfes}/{annots}.pdf',
I know it is huge, but I tend to think that this is better than having the same name across different simulation types.

Reply:
right now everything is sorted by directory. we could change this later if needed